### PR TITLE
Ajax recommended songs

### PR DESCRIPTION
**Pull request by Mycobee**
Adds jQuery ajax request on dashboard path

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,4 +12,34 @@
 //
 //= require rails-ujs
 //= require activestorage
+//= require jquery3
+//= require jquery_ujs
 //= require_tree .
+//  var dataid = $(this).attr('data-id');
+//  var id = $(this).closest('.test').attr('id');
```
Comment (WHomer) [inline app/assets/javascripts/application.js]: can we remove tis commented code?

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -12,4 +12,34 @@
...
+     data : params,
+     dataType: "json",
+     success: function (new_recommended) {
+				var i 
+				for (i = 0; i < 5; i++) {
+					$('.recommended-iframe')[i].src = 'https://open.spotify.com/embed/track/' + new_recommended[i].spotify_id
+				}
+				//for var_i = 0 to new_recommended.length
+
+				//set the song of each source by index i
```
Comment (WHomer) [inline app/assets/javascripts/application.js]: do we still need these logs?

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -12,5 +12,25 @@
 //
 //= require rails-ujs
 //= require activestorage
+//= require jquery3
+//= require jquery_ujs
 //= require_tree .
-//= require Chart.bundle.min
```
Comment (BrennanAyers) [inline app/assets/javascripts/application.js]: @WHomer Hey let's add this back in, should be good.

Comment (WHomer) [reply app/assets/javascripts/application.js]: Yeah, I think that would be a great idea.  Let me consult the QA team to see how this was missed in the first place.

Comment (BrennanAyers) [reply app/assets/javascripts/application.js]: @WHomer Addressed in #66.

**Pull request merged**